src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,42 @@
-export const PeopleFilters = () => {
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Link } from 'react-router-dom';
+import { SortSex } from '../types/SortSex';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  centuries: string[];
+  query: string;
+  sex: string;
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
Change the type of 'sex' prop from string to SortSex.

Reply:
Thank you for such a detailed review of my code 🥰.